CHANGELOG.md

```diff
@@ -4,6 +4,35 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.0-rc1 - 2022-10-12
```

Comment:
The release notes are not final, correct? Can we still edit them after merging to main? We will have more stuff in here to expand on. I would add most of the feature flags in more detail.

Reply:
correct -- even though I'm merging this in -- I do not intend to release it as is.